psalm-baseline.xml

```diff
@@ -149,16 +149,12 @@
       <code><![CDATA[getStartLine]]></code>
       <code><![CDATA[getStmts]]></code>
       <code><![CDATA[getSubNodeNames]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
-      <code><![CDATA[hasAttribute]]></code>
       <code><![CDATA[traverse]]></code>
     </ImpureMethodCall>
+    <PropertyTypeCoercion>
```

Comment:
these errors are related to https://github.com/Roave/BetterReflection/pull/1407#issuecomment-1996573163